### PR TITLE
AP-151

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

**Which issue(s) this PR fixes** 🔨

AP-151

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by Jekrimo — APPROVED** [review]

**Review by stavares843 — CHANGES_REQUESTED** [review]
Captura de ecrã 2021-12-06, às 22 23 08

screen has the chat bar, can we add the proper string instead of showing `global.talk` there?

File: components/tailored/core/chatbar/Chatbar.vue [hunk 1]
```diff
@@ -23,10 +23,6 @@ declare module 'vue/types/vue' {
     handleInputChange: Function
     value: string
     updateText: Function
-    handleUpload: Function
-    debounceTypingStop: Function
```
Comment (WanderingHogan) [inline components/tailored/core/chatbar/Chatbar.vue:27]: Can you add these back in - they allow for the typescript code hints/without them we get the typescript is angry red squiggles

File: components/tailored/core/chatbar/Chatbar.vue [hunk 2]
```diff
@@ -39,8 +35,7 @@ export default Vue.extend({
     return {
       text: '',
       showEmojiPicker: false,
-      maxChars: 256,
-      recipientTyping: false,
```
Comment (WanderingHogan) [inline components/tailored/core/chatbar/Chatbar.vue:43]: we need this back in

File: components/tailored/core/chatbar/controls/Controls.html [hunk 3]
```diff
@@ -14,11 +15,12 @@
   </div>
   <div
     id="glyph-toggle"
-    :data-tooltip="$t('ui.glyphs')"
+    :data-tooltip="$t('global.glyphs')"
```
Comment (WanderingHogan) [inline components/tailored/core/chatbar/controls/Controls.html:18]: Where is global coming from? When I look at `https://github.com/Satellite-im/locales/blob/main/en-US.js` - I think these should be set back to ui

File: components/tailored/core/chatbar/Chatbar.vue (hunk 2 above)
Comment (davidchan1219) [reply components/tailored/core/chatbar/Chatbar.vue]: I added it

File: components/tailored/core/chatbar/controls/Controls.html (hunk 3 above)
Comment (davidchan1219) [reply components/tailored/core/chatbar/controls/Controls.html]: I updated it

File: components/tailored/core/chatbar/controls/Controls.html [hunk 4]
```diff
@@ -27,22 +29,25 @@
...
     id="emoji-toggle"
-    :data-tooltip="$t('ui.emotes')"
+    :data-tooltip="$t('global.emotes')"
     v-on:click="toggleEnhancers('emotes')"
-    class="
-      control-tooltip-wrap
+    :class="
+      `control-tooltip-wrap
       has-tooltip has-tooltip-top has-tooltip-primary
+      ${editable ? 'active' : 'inactive'}`
```
Comment (WanderingHogan) [inline components/tailored/core/chatbar/controls/Controls.html:37]: Can you make this
class="control-tooltip-wrap  has-tooltip has-tooltip-top has-tooltip-primary"
:class="editable ? 'active' : 'inactive'", and make the the other v-bind classes in this file  also split like this

File: components/tailored/core/fileupload/FileUpload.html [hunk 5]
```diff
@@ -58,12 +58,12 @@
...
   </div>
-  <input type="file" id="quick-upload" @change="handleFile" />
+  <input :type="editable ? `file` : `hidden`" id="quick-upload" @change="handleFile" />
   <label
     v-if="type == 'quick'"
     for="quick-upload"
-    :data-tooltip="$t('controls.upload')"
-    class="has-tooltip has-tooltip-top has-tooltip-primary upload-label"
+    :data-tooltip="$t('global.upload')"
+    :class="`has-tooltip has-tooltip-top has-tooltip-primary upload-label ${editable ? 'active' : 'inactive'}`"
```
Comment (WanderingHogan) [inline components/tailored/core/fileupload/FileUpload.html:66]: same as in controls.html

File: components/tailored/core/fileupload/FileUpload.vue [hunk 6]
```diff
@@ -46,27 +49,23 @@ export default Vue.extend({
...
+        if (files.length > 4) {
+          // @ts-ignore
+          this.$data.count_error = true
+          return
         }
-      })
-      /* nsfw checking after putting all files */
-      for (const file of this.$data.files) {
-        // don't check nsfw for large files or webgl runs out of memory
-        if (file.file.size > Config.uploadByteLimit) {
```
Comment (WanderingHogan) [inline components/tailored/core/fileupload/FileUpload.vue:66]: Were these couple lines removed on purpose?

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
Good start, please check comments when you get a chance

File: components/tailored/core/fileupload/FileUpload.html (hunk 5 above)
Comment (davidchan1219) [reply components/tailored/core/fileupload/FileUpload.html:66]: I updated it

File: components/tailored/core/chatbar/controls/Controls.html (hunk 4 above)
Comment (davidchan1219) [reply components/tailored/core/chatbar/controls/Controls.html]: I made classes

File: components/tailored/core/fileupload/FileUpload.vue (hunk 6 above)
Comment (davidchan1219) [reply components/tailored/core/fileupload/FileUpload.vue:66]: I reverted it on AP-151 branch

**Comment by WanderingHogan** [comment]
Per the DM - please put the changes for this ticket in a new pull request

**Pull request closed without merging**